streampark-console/streampark-console-webapp/src/views/flink/app/hooks/useCreateAndEditSchema.ts

```diff
@@ -466,8 +467,8 @@ export const useCreateAndEditSchema = (
         field: 'yarnQueue',
         label: t('flink.app.yarnQueue'),
         component: 'Input',
-        componentProps: { placeholder: t('flink.app.addAppTips.yarnQueuePlaceholder') },
         ifShow: ({ values }) => values.executionMode == ExecModeEnum.YARN_APPLICATION,
```

Comment:
both `ExecModeEnum.YARN_APPLICATION ` and `ExecModeEnum.YARN_PER_JOB` need to be show